src/polkadotApi/PolkadotApiWebView.tsx

```diff
@@ -345,7 +345,6 @@ function useWebViewOnMessage(resolversRef: ResolversRef, postMessage: PostMessag
 
   const webViewOnMessage = React.useCallback(
     (event: WebViewMessageEvent) => {
-      console.info('WebView Response: ', event.nativeEvent.data);
```

Comment:
You can wrap this into `(__DEV__) {...}` but let's keep it. It's quite helpful when developing